soap/src/java/com/zimbra/soap/mail/type/FreeBusySlot.java

```diff
@@ -40,6 +40,69 @@
      */
     @XmlAttribute(name=MailConstants.A_CAL_END_TIME /* e */, required=true)
     private final long endTime;
+    
+    /**
+     * @zm-api-field-tag id
+     * @zm-api-field-description calendar event id
+     */
+    @XmlAttribute(name=MailConstants.E_CAL_EVENT_ID, required=false)
+    private String id;
+    
+    /**
+     * @zm-api-field-tag subject
+     * @zm-api-field-description Appointment subject
+     */
+    @XmlAttribute(name=MailConstants.E_CAL_EVENT_SUBJECT, required=false)
+    private String subject;
+    
+    /**
+     * @zm-api-field-tag location
+     * @zm-api-field-description location of meeting
+     */
+    @XmlAttribute(name=MailConstants.E_CAL_EVENT_LOCATION, required=false)
+    private String location;
+    
+    /**
+     * @zm-api-field-tag isMeeting
+     * @zm-api-field-description returns a boolean value whether this calendar event is a meeting or not.
+     */
+    @XmlAttribute(name=MailConstants.E_CAL_EVENT_ISMEETING, required=false)
+    private boolean isMeeting;
+    
+    /**
+     * @zm-api-field-tag isRecurring
+     * @zm-api-field-description returns a boolean indicating whether it is continuous or not.
```

Comment:
instead of continuous use recurring.
```suggestion
     * @zm-api-field-description returns a boolean indicating whether it is recurring or not.
```